source/_components/brottsplatskartan.markdown

```diff
@@ -2,15 +2,15 @@
 layout: page
 title: "Brottsplatskartan"
 description: "Instructions on how to integrate brottsplatskartan.se into Home Assistant."
-date: 2018-12-02 20:00
+date: 2019-05-11 22:30
```

Comment:
Please do not change this

Reply:
Thanks, changed it back to the old value.